### PR TITLE
Create ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by ensu

Ticket: [MTM-50050](https://cumulocity.atlassian.net/browse/MTM-50050)
Original PR: [5212](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5212)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Added alert when MO delete operation is taking more time.
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Add alert for long-running managed object delete operations
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-50050
+version: 1020.0.0
+---
+Added alert when MO delete operation is taking more time.
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md:17]: The Claude AI suggested to use the following body:
```suggestion
In some situations, deleting managed objects in Cumulocity IoT could take an unexpectedly long time without providing any feedback to the user. To improve visibility, an alert is now shown in the UI when a delete operation exceeds a certain duration. This allows users to track the progress of long-running deletions. The alert is automatically dismissed once the operation completes successfully.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md (hunk 1 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md:3]:
```suggestion
title: An alert is shown for long-running managed object operations.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md (hunk 2 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md:17]:
```suggestion
In some situations, managed object operations in Cumulocity IoT could take an unexpectedly long time without providing any feedback to the user. To improve visibility, an alert is now shown in the UI when a certain operation exceeds a specific duration.
```

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-50050
+version: 1020.0.0
+---
+In some situations, managed object operations in Cumulocity IoT could take an unexpectedly long time without providing any feedback to the user. To improve visibility, an alert is now shown in the UI when a certain operation exceeds a specific duration.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md:17]:
```suggestion
In some situations, managed object operations in Cumulocity IoT could take an unexpectedly long time without providing any feedback to the user. For more transparency, an alert is now shown in the UI when a certain operation exceeds a specific duration.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: An alert is shown for long-running managed object operations.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-added-alert-when-mo-delete-operation-is-taking-more-time.md:3]:
```suggestion
title: Alert for long-running managed object operations
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@rahultsag Please review and approve

**Review by rahultsag — APPROVED** [review]

**Pull request merged**